search/tests/test_sorting.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
And I totally disagree with  "it doesn't do anything for test coverage."
coverage.py just tests that a piece of code is actually called. The fact that it works as expected is not tested.